webpack.config.js

```diff
@@ -137,9 +137,9 @@ module.exports = (env) => {
 
     if (env === 'production') {
         return [
-            libraryESMConfig,
+            // libraryESMConfig,
             libraryUMDConfig,
-            bundleConfig,
+            // bundleConfig,
```

Comment:
ESM build currently doesn't work correctly. Bundleconfig isn't used ATM so commenting it out for now for quicker builds.